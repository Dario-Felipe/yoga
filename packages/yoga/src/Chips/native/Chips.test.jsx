import React from 'react';
import { render } from '@testing-library/react-native';

import { ThemeProvider, Chips } from '../..';

describe('<Chips />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider>
        <Chips />
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
