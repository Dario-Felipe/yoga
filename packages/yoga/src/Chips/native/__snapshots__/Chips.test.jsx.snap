// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chips /> Snapshots counter should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback
    counter={8}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;

exports[`<Chips /> Snapshots counter should match snapshot with more than one Chip 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback
    counter={8}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    counter={52}
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Gyms and studios
        </Text>
        <Text
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#D8385E",
                "borderRadius": 8,
                "color": "#FFFFFF",
                "display": "flex",
                "flexShrink": 0,
                "fontFamily": "Rubik",
                "fontSize": 12,
                "fontWeight": "500",
                "height": 16,
                "justifyContent": "center",
                "lineHeight": 16,
                "marginLeft": 4,
                "minWidth": 16,
                "paddingLeft": 5,
                "paddingRight": 5,
              },
            ]
          }
        >
          52
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    counter={1732}
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Personal trainers
        </Text>
        <Text
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#D8385E",
                "borderRadius": 8,
                "color": "#FFFFFF",
                "display": "flex",
                "flexShrink": 0,
                "fontFamily": "Rubik",
                "fontSize": 12,
                "fontWeight": "500",
                "height": 16,
                "justifyContent": "center",
                "lineHeight": 16,
                "marginLeft": 4,
                "minWidth": 16,
                "paddingLeft": 5,
                "paddingRight": 5,
              },
            ]
          }
        >
          +999
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;

exports[`<Chips /> Snapshots icon should match selected snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback
    counter={8}
    icon={
      Array [
        [Function],
      ]
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
          ]
        }
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Location
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    counter={8}
    icon={
      Array [
        [Function],
        [Function],
      ]
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
            [Function],
          ]
        }
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    icon={
      Array [
        [Function],
        [Function],
      ]
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
            [Function],
          ]
        }
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Gyms and studios
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    icon={
      Array [
        [Function],
      ]
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
          ]
        }
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Personal trainers
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    counter={8}
    icon={
      Array [
        [Function],
      ]
    }
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
          ]
        }
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Activities
        </Text>
        <Text
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#D8385E",
                "borderRadius": 8,
                "color": "#FFFFFF",
                "display": "flex",
                "flexShrink": 0,
                "fontFamily": "Rubik",
                "fontSize": 12,
                "fontWeight": "500",
                "height": 16,
                "justifyContent": "center",
                "lineHeight": 16,
                "marginLeft": 4,
                "minWidth": 16,
                "paddingLeft": 5,
                "paddingRight": 5,
              },
            ]
          }
        >
          8
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    counter={8}
    icon={
      Array [
        [Function],
        [Function],
      ]
    }
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
            [Function],
          ]
        }
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Language
        </Text>
        <Text
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#D8385E",
                "borderRadius": 8,
                "color": "#FFFFFF",
                "display": "flex",
                "flexShrink": 0,
                "fontFamily": "Rubik",
                "fontSize": 12,
                "fontWeight": "500",
                "height": 16,
                "justifyContent": "center",
                "lineHeight": 16,
                "marginLeft": 4,
                "minWidth": 16,
                "paddingLeft": 5,
                "paddingRight": 5,
              },
            ]
          }
        >
          8
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    icon={
      Array [
        [Function],
        [Function],
      ]
    }
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
            [Function],
          ]
        }
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Categories
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    icon={
      Array [
        [Function],
      ]
    }
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        icon={
          Array [
            [Function],
          ]
        }
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Availability
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;

exports[`<Chips /> Snapshots selected should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;

exports[`<Chips /> Snapshots selected should match snapshot with more than one Chip 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Gyms and studios
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback
    selected={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFEEF2",
              "borderColor": "#D8385E",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          selected={true}
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "700",
              },
              Object {
                "color": "#D8385E",
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Personal trainers
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;

exports[`<Chips /> Snapshots should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;

exports[`<Chips /> Snapshots should match snapshot with disabled prop 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback
    disabled={true}
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={true}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#F5F5FA",
              "borderColor": "#D7D7E0",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "color": "#9898A6",
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;

exports[`<Chips /> Snapshots should match snapshot with more than one Chip 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <TouchableWithoutFeedback>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Classes
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Gyms and studios
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
  <TouchableWithoutFeedback>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        disabled={false}
        pressed={false}
        selected={false}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 8,
              "borderStyle": "solid",
              "borderWidth": 1,
              "display": "flex",
              "flexDirection": "row",
              "height": 32,
              "justifyContent": "center",
              "marginRight": 16,
              "maxWidth": 216,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
            },
          ]
        }
      >
        <Text
          inverted={false}
          light={false}
          selected={false}
          size="medium"
          style={
            Array [
              Object {
                "color": "#231B22",
                "fontFamily": "Rubik",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
              Object {
                "fontSize": 12,
                "lineHeight": 16,
              },
            ]
          }
        >
          Personal trainers
        </Text>
      </View>
    </View>
  </TouchableWithoutFeedback>
</View>
`;
